packages/netglade_utils/lib/src/extensions/string_extensions.dart

```diff
@@ -51,6 +51,31 @@ extension StringExtensions on String {
 
     return replaceAll(pattern, '');
   }
+
+  String capitalize() {
+    return '${this[0].toUpperCase()}${characters.getRange(1).toLowerCase()}';
```

Comment:
Je tohle safe? Co kdyz se to zavola na prazdnym stringu?

Reply:
Pravda, opraveno

Reply:
Opraveno